terraform/README.md

```diff
@@ -0,0 +1,132 @@
+# Terraform Infrastructure for Team Growth ETL Pipeline
+
+This repository contains Terraform configurations to set up the infrastructure required for the `c14-team-growth` project. It provisions and manages resources on AWS, including ECS, Lambda, IAM roles, S3, and supporting services.
+
+---
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [Infrastructure Components](#infrastructure-components)
+   - [Providers](#providers)
+   - [Security Groups](#security-groups)
+   - [ECR Repository](#ecr-repository)
+   - [IAM Roles and Policies](#iam-roles-and-policies)
+   - [ECS Task Definition](#ecs-task-definition)
+   - [CloudWatch Log Groups](#cloudwatch-log-groups)
+   - [Lambda](#lambda)
+   - [EventBridge Scheduler](#eventbridge-scheduler)
+   - [S3 Bucket](#s3-bucket)
+3. [How It Works](#how-it-works)
+4. [Usage](#usage)
+5. [Additional Notes](#additional-notes)
+
+---
+
+## Overview
+
+This Terraform setup provisions an AWS infrastructure to support an ETL pipeline for transferring data from RDS to S3 and running scheduled tasks. Key services include:
+
+- **ECS (Elastic Container Service)**: Executes the ETL process as a Fargate task.
+- **Lambda**: Handles additional processing tasks triggered by EventBridge.
+- **S3**: Stores long-term data.
+- **EventBridge**: Manages scheduled execution of tasks and functions.
+- **CloudWatch**: Captures logs for monitoring.
+
+---
+
+## Infrastructure Components
```

Comment:
Looking great, reminder this will need to be updated to reflect the dashboard living on AWS. Will likely involve;
Another ECR
ECS Service

Reply:
Will update when we have it added.